### PR TITLE
Various Suggestions

### PR DESCRIPTION
**Pull request by djzort**
A range of suggestions with their own comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,6 +217,7 @@ We also recommend these lists.
 
 *Libraries and tools for templating*
 
+* [Text;;Templates](https://metacpan.org/pod/Text::Template) - Templates with embedded perl
```
Comment (moznion) [inline README.md]: Here is typo :smile:

**Comment by moznion** [comment]
I do not have confidence, is `Data::Dumper` appropriate for the member of `Data Format` section?
If you have another good idea, please feel free to tell me.

File: README.md (hunk 1 above)
Comment (djzort) [inline README.md]: I have updated this typo in ba92477

**Comment by djzort** [comment]
Perhaps Data::Dumper and its work-alikes deserve their own section?

**Comment by moznion** [comment]
Probably so i feel.

**Comment by moznion** [comment]
Okay, I'll merge for the present!

**Pull request merged**